### PR TITLE
Correctly pass Storyblok version when fetching story

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

- Fix passing default Storyblok version when fetching story data

- Update logging when building CMS pages

## Justify why they are needed

Fix issue in the build pipeline -- we incorrectly default to "draft" mode.

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #2674** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2674" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2674?utm_source=stack-comment).

File: apps/store/src/services/storyblok/storyblok.ts [hunk 1]
```diff
@@ -332,7 +332,7 @@ export const getStoryBySlug = <StoryData extends ISbStoryData>(
   { version, locale }: StoryOptions,
 ): Promise<StoryData> => {
   const params: StoryblokFetchParams = {
-    version: version ?? USE_DRAFT_CONTENT ? 'draft' : 'published',
+    version: version ?? (USE_DRAFT_CONTENT ? 'draft' : 'published'),
```
Comment (robinandeer) [inline apps/store/src/services/storyblok/storyblok.ts:335]: This line was evaluated as `(version ?? USE_DRAFT_CONTENT) ? 'draft' : 'published'` which is not what I intended....

Comment (alebedev) [reply apps/store/src/services/storyblok/storyblok.ts:335]: nesting ternaries with other expressions is evil :)

**Review by alebedev — APPROVED** [review]
Nice catch

**Pull request merged**